Description: fix missing .tarball-version if git isn't used
Author: Jörg Frings-Fürst <debian@jff.email>
Bug: https://gitlab.com/sane-project/backends/-/issues/440
Last-Update: 2021-02-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trunk/.tarball-version
===================================================================
--- /dev/null
+++ trunk/.tarball-version
@@ -0,0 +1 @@
+1.0.32-debian
